kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/players/QueuedAudioPlayer.kt

```diff
@@ -73,11 +73,9 @@ class QueuedAudioPlayer(context: Context, bufferConfig: BufferConfig? = null) :
     /**
      * Add a single item to the queue.
      * @param item The [AudioItem] to add.
-     * @param playWhenReady If this is `true` it will automatically start playback. Default is `true`.
+     * @param playWhenReady If the AudioPlayer has no item loaded, it will load the `item`. If this is `true` it will automatically start playback. Default is `true`.
```

Comment:
The sentence you added doesn't seem related to the `playWhenReady` param? Maybe it should be at the top.

Reply:
it kind of is, but either way sounds good. I'll update it.